### DISCUSSION TITLE
launched Lift Hill Corkscrew Coaster And Hyper Coaster

### DISCUSSION THREAD

**Discussion by Coasterhacker** [OpenRCT2/OpenRCT2#21660] [Feature requests]
![Six Flags 2024-03-23 20-58-27](https://github.com/OpenRCT2/OpenRCT2/assets/155395270/36433fe4-a83f-446e-b03f-979c7db6a641)very strange. but they can also work on these two roller coasters. is it possible to get that speed lift hill for those two roller coasters

**Comment by Realsteel89** [comment]
Do you had an example for an real track like this?

**Reply by Coasterhacker** [reply]
you must need good image to show. is this image good enough for you. I had two made

**Reply by karst** [reply]
Do you have an example of the rides in actual existing parks that use boosters on these models?

**Reply by Coasterhacker** [reply]
Can we get back to my question? is this possible in the game

**Comment by Coasterhacker** [comment]


**Reply by Coasterhacker** [reply]
I've been making this fuss for a while now. and sometimes it is extreme and sometimes it is just right. in terms of test results. depends on how you make your coaster

**Reply by Coasterhacker** [reply]
it will be crazy that corkscrew coaster and hypercoaster have a launched lift hill. you know they accelerate faster. and I will like having them in the game. now i use cheat or plugins to do it.

**Comment by Coasterhacker** [comment]
![Test Coaster and CTR 2024-03-26 19-05-50](https://github.com/OpenRCT2/OpenRCT2/assets/155395270/7e542fd5-333a-4d38-94e5-abbc168c4a42)
i think lim is better than tires. lim seems much more modern. but can this be done in the game without cheat or plugins. it is possible or is that too much to ask

**Comment by Coasterhacker** [comment]
I don't think Real Park has it. I thought roller coasters had this. I think Vekoma might have it. and if not. I still really enjoy coming into the game🙏

**Reply by Coasterhacker** [reply]
I have done research. I have good news. hyperspace mountain disneyland paris has launched lift hill

**Reply by Realsteel89** [reply]
It Will be super easy to implement this

**Reply by Realsteel89** [reply]
It Will be super easy to implement this ~~~~

**Reply by Coasterhacker** [reply]
so it is possible. If this is included in the update, I am very happy that I no longer have to use cheats and plugins. i use cheat and plugins to do

**Reply by Realsteel89** [reply]
Every add on tracks  is possible the import self from the code is easy doing because it exist on other rides. You only need to add the code with copy paste and change the variables into the code.

**Reply by Coasterhacker** [reply]
does that mean I have to do that or you have to. i don't know anything about code.sorry i don't know anything about code or how to do it.

**Reply by Coasterhacker** [reply]
I'll let the developers or someone do that.

**Reply by Realsteel89** [reply]
Someone made an branch for it and develop the code and sprite than wait to merge this in the develop branch

**Reply by Coasterhacker** [reply]
Okay. so someone does it then.

**Reply by Coasterhacker** [reply]
I'll see it in the next update
